overlord/configstate/configcore/timezone.go

```diff
@@ -95,3 +108,23 @@ func handleTimezoneConfiguration(_ sysconfig.Device, tr config.ConfGetter, opts
 
 	return nil
 }
+
+func getTimezoneFromSystemVC(key string) (interface{}, error) {
+	return getTimezoneFromSystem()
+}
+
+func getTimezoneFromSystem() (string, error) {
+	// We cannot use "timedatectl show" here because it is only
+	// available on UC20
+	link, err := os.Readlink(filepath.Join(dirs.GlobalRootDir, "/etc/writable/localtime"))
+	// see localtime(5)
+	// "If /etc/localtime is missing, the default "UTC" timezone is used."
```

Comment:
Do we expect to have `/etc/writable/localtime` in all systems? In other words, don't we need to fall back and check also `/etc/localtime`?

(if not, the code is correct, but the comment about `/etc/localtime` could be expanded to clarify that)

Reply:
I added a comment - this code will only run on Ubuntu Core systems which all have /etc/writable. Never on classic.

Reply:
Thanks!